fineract-provider/src/main/resources/application.properties

```diff
@@ -98,6 +98,7 @@ fineract.remote-job-message-handler.kafka.admin.extra-properties-key-value-separ
 fineract.remote-job-message-handler.kafka.admin.extra-properties=${FINERACT_REMOTE_JOB_MESSAGE_HANDLER_KAFKA_ADMIN_EXTRA_PROPERTIES:}
 
 fineract.events.external.enabled=${FINERACT_EXTERNAL_EVENTS_ENABLED:false}
+fineract.events.external.partition-size=${FINERACT_EXTERNAL_EVENTS_PARTITION_SIZE:5000}
```

Comment:
Please dont forget to add this field to the application-test.properties as well ;)

Reply:
done